content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -5,15 +5,36 @@ reviewers:
 - erictune
 - krousey
 - clove
+content_template: templates/concept
+toc_hide: false
```

Comment:
nit: I think this is the default so you can remove it, but it probably won't hurt.

Reply:
Yes, i just tested it and it is default. I am removing it.